src/app/(admin)/new_dashboard/guides/hooks/useSearchGuides.ts

```diff
@@ -1,18 +1,25 @@
 import { useState } from "react";
 import { Guide } from "@/api/types/Guide";
+import { SelectChangeEvent } from "@mui/material";
+import { SortCriteria } from "../types";
+import { sortByCriteria } from "./sortGuides";
 
 export const useSearchGuides = (initialGuides: Guide[]) => {
   const [searchQuery, setSearchQuery] = useState("");
   const [selectedTag, setSelectedTag] = useState("All");
+  const [sortCriteria, setSortCriteria] = useState<SortCriteria>("rating");
 
-  const filteredGuides = initialGuides.filter((guide) => {
-    const matchesQuery = guide.title
-      .toLowerCase()
-      .includes(searchQuery.toLowerCase());
-    const matchesTag =
-      selectedTag === "All" || guide.categories.includes(selectedTag);
-    return matchesQuery && matchesTag;
-  });
+  const filteredGuides = sortByCriteria(
+    initialGuides.filter((guide) => {
```

Comment:
dont pass it directly into the function, make a const of filtered guides and then pass the const to the function

Reply:
ok